### PR TITLE
Update feedback section

### PR DESCRIPTION
**Pull request by lukebrooker**
image

### CODE REVIEW

File: packages/experience-docs/source/components/Icon/index.js [hunk 1]
```diff
@@ -14,12 +14,15 @@ const getIcon = (icon) => find(ICONS, ({title}) => title === startCase(icon))
...
-  kind = 'flat'
+  kind = 'flat',
+  ...props
 }) => {
   const {src, title} = getIcon(icon)
-  return src && <SVGIcon kind={kind} svg={src} title={title} alt={title} />
+  return src && <SVGIcon {...props} kind={kind} svg={src} title={title} alt={title} />
 }
 
+Icon.isStrange = true
```
Comment (benoneal) [inline packages/experience-docs/source/components/Icon/index.js]: this looks like a hack...

File: packages/experience-docs/source/components/Icon/index.js [hunk 2]
```diff
@@ -14,12 +14,15 @@ const getIcon = (icon) => find(ICONS, ({title}) => title === startCase(icon))
 const Icon = ({
   name,
   icon,
-  kind = 'flat'
+  kind = 'flat',
+  ...props
 }) => {
   const {src, title} = getIcon(icon)
-  return src && <SVGIcon kind={kind} svg={src} title={title} alt={title} />
+  return src && <SVGIcon {...props} kind={kind} svg={src} title={title} alt={title} />
```
Comment (benoneal) [inline packages/experience-docs/source/components/Icon/index.js]: I'd really rather we avoid ever doing this. This basically says "pass anything to me and I'll pass it on". You lose all visibility of how this is being used and what you can expect to be passed to child components, so you can get unpredictable behaviour. If a component renders a child that requires a prop, that component must also require that prop explicitly.

File: packages/experience-docs/source/components/Feedback/index.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  ' h2': {
+    fontSize: font.scale(2),
+    marginBottom: size(4)
+  },
+  ' ul': {
+    marginBottom: size(5)
+  }
+}))(Prose)
+
+const Feedback = ({ props }) => {
```
Comment (benoneal) [inline packages/experience-docs/source/components/Feedback/index.js]: If props isn't used, don't declare it in the args. That way it's explicit and obvious to anyone else looking at this component that it doesn't have any customizable interface.

File: packages/experience-docs/source/components/Icon/index.js (hunk 2 above)
Comment (lukebrooker) [reply packages/experience-docs/source/components/Icon/index.js]: Fair enough. I did this quickly. I had always planned to explicitly add margin props.

File: packages/experience-docs/source/components/Feedback/index.js (hunk 3 above)
Comment (lukebrooker) [reply packages/experience-docs/source/components/Feedback/index.js]: Agree. This was from old code. Thanks.

File: packages/stranger/src/things/component.js [hunk 4]
```diff
@@ -17,9 +17,10 @@ const comp = (styles = {}) => (Component = 'div') => {
       ? merge({}, stylesObj, existingStyles)
       : stylesObj
     const className = addRule(mergedStyles)
+    const newProps = { ...rest, styles: mergedStyles }
     const propsToPass = Component.isStrange
-      ? { ...rest, styles: mergedStyles }
-      : pickBy(rest, validAttr(Component.propTypes))
+      ? newProps
```
Comment (HowlingEverett) [inline packages/stranger/src/things/component.js]: This feels like a bugfix on Stranger's comp component. Split it out into another branch and PR it separately so we can have a changelog in the future? (Sorry, am all about doccing right now)

Comment (lukebrooker) [reply packages/stranger/src/things/component.js]: Sounds good

**Comment by HowlingEverett** [comment]
Otherwise looks good to me 👍 .

**Pull request merged**